passage/chain.json

```diff
@@ -1,7 +1,8 @@
 {
   "$schema": "../chain.schema.json",
   "chain_name": "passage",
-  "chain_id": "passage-1",
+  "chain_id": "passage-2",
+  "pre_fork_chain_name": "passage-1",
```

Comment:
I'd say pre_frok_chain_name would be passage1; i.e., the name, not the id
If used, it should be added to the schema

In your passage1 branch, I'd change the chain_name to 'passage1'

Reply:
https://github.com/cosmos/chain-registry/pull/2598